helper/hex/hex_test.go

```diff
@@ -0,0 +1,42 @@
+package hex
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestEncodingUint64(t *testing.T) {
+	testTable := []struct {
+		name       string
+		value      uint64
+		shouldFail bool
```

Comment:
Why have this in the test if none of your test cases fail?

Reply:
fixed 8ef3965ab7c423466a31342f16a9715bd53826b6